databook/background/Two-photon-calcium-imaging.md

```diff
@@ -52,21 +56,23 @@ brains. While wide-field microscopy can still be used on three-dimensional
+two photons to excite. {cite:p}`denk1990`. The primary resulting photon emission mode
+from the molecule as it de-excites is thus at a wavelength significantly smaller that of the laser,
+so the stimulating light does not interfere with the signal. Moreover, with two-photon excitation,
+the absorption is a nonlinear function of photon density, allowing microscopes to
+only excite a single volume pixel (voxel) of tissue at a time by adjusting the focal point of the laser. 
+This improves our ability to image the activity of {term}`GECI` expressing neurons in intact tissue.
+
+There are limits to what we can see using two-photon microscopy, however. Two-photon excitation generates fluorescence primarily from the focal plane, but out-of-focus fluorescence from outside the focal plane will contaminate the view. This out-of-focus fluorescence increases as the focus is pushed deeper into tissue {cite:p}`takasaki`. This can be problematic when imaging densely expressed {term}`GECI`s where there is fluorescence throughout the tissue, causing the image quality to degrade with imaging depth. However, using {term}`Cre line`s that limit the GECI expression only to deep neurons (e.g. Cre lines specific to layer 5 or layer 6 {term}`pyramidal cell`s) can avoid this problem.
 
 ![2P3P](/resources/2P3P.jpg)
 
 ## Event detection
 
-The spatial and temporal resolution of the imaging conditions determine the ability to resolve individual, or even small numbers of, spikes. Validation datasets, in which individual neurons are imaged using 2-photon calcium imagine at the same time as intracellular voltage recordings are made using patch clamp techniques, usually collect data with high spatial and temporal resolution. At these resolutions, single action potentials are discernable from the fluoresence traces. However, population recordings like the ones in our Visual Coding 2-photon and Visual Behavior Ophys datasets, are routinely made with lower spatial and temporal resolutions. This allows us to image large populations of neurons. But, the result is an inability to reliably resolve low firing rate events {cite:p}`huang`.
+The spatial and temporal resolution of the imaging conditions determine the ability to resolve individual, or even small numbers of, spikes. Validation datasets, in which individual neurons are imaged using 2-photon calcium imagine at the same time as intracellular voltage recordings are made using patch clamp techniques, usually collect data with high spatial and temporal resolution. At these resolutions, single action potentials are discernable from the fluorescence traces. However, population recordings like the ones in our [Visual Coding 2-photon](vc2p-background) and [Visual Behavior Ophys](VB-Ophys) datasets, are routinely made with lower spatial and temporal resolutions. This allows us to image large populations of neurons. But, the result is an inability to reliably resolve low firing rate events {cite:p}`huang`.
```

Comment:
let's just say "in our ophys [link to ophys page] datasets"  since it is true of all our ophys datasets